widget/testdata/accordion/layout_expanded_multiple_open_multiple_items.xml

```diff
@@ -8,15 +8,15 @@
 					<widget pos="32,8" size="9x20" type="*widget.RichText">
 						<text alignment="center" bold size="9x19">A</text>
 					</widget>
-					<image fillMode="contain" pos="8,8" rsc="menuDropDownIcon" size="iconInlineSize" themed="foreground"/>
```

Comment:
AFAICS, this (and other) previously were explicitly themed with the (default) foreground color. Now, it says that it is no longer themed anymore and thus will use the image’s colors, right? Is this really wanted?

Reply:
There was inconsistency before - named resources did not seem to have theme information (at least not some of them?) for example this happened on one test:
```
								<image pos="66,7" rsc="menuExpandIcon" size="iconInlineSize"/>
								<image pos="8,7" rsc="account.svg" size="iconInlineSize" themed="default"/>
```

I have updated the markup test renderer to be more consistent which has added changes. A smaller change set than before which seems promising - but overall it is now explicit about what theme is applied on all resources I think...